ts/parts-map/HeatmapSeries.ts

```diff
@@ -293,6 +320,28 @@ seriesType<Highcharts.HeatmapSeries>(
             options.pointRange = pick(options.pointRange, options.colsize || 1);
             // general point range
             this.yAxis.axisPointRange = options.rowsize || 1;
+
+            // Bind new symbol names
+            extend(symbols, {
+                ellipse: symbols.circle,
+                rect: symbols.square
+            });
+        },
+        getSymbol: seriesTypes.scatter.prototype.getSymbol,
+
+        /**
+         * @private
+         * @function Highcharts.seriesTypes.heatmap#setClip
+         * @return {void}
+         */
+        setClip: function (
+            this: Highcharts.HeatmapSeries
+        ): void {
+            Series.prototype.setClip.apply(this, arguments);
+            if (this.options.clip !== false && this.markerGroup) {
+                this.markerGroup
+                    .clip(this.chart.clipRect);
```

Comment:
I think `this.sharedClipRect` should be used (if defined). Otherwise x/yAxis top/left/height/width can clip incorrectly?